v3/src/lib/use-cloud-file-manager.ts

```diff
@@ -2,14 +2,15 @@ import { CFMAppOptions, CloudFileManager, CloudFileManagerClientEvent } from "@c
 import { useEffect, useRef } from "react"
 import { Root, createRoot } from "react-dom/client"
 import { useMemo } from "use-memo-one"
-import { clientConnect, createCloudFileManager, renderRoot } from "./cfm-utils"
-import { handleCFMEvent } from "./handle-cfm-event"
+import { codapResourcesUrl } from "../constants"
 import { appState } from "../models/app-state"
 import { isCodapDocument } from "../models/codap/create-codap-document"
 import { gLocale } from "../utilities/translation/locale"
 import { t } from "../utilities/translation/translate"
 import { removeDevUrlParams, urlParams } from "../utilities/url-params"
+import { clientConnect, createCloudFileManager, renderRoot } from "./cfm-utils"
 import { DEBUG_CFM_LOCAL_STORAGE } from "./debug"
+import { handleCFMEvent } from "./handle-cfm-event"
```

Comment:
Hmm. I guess we aren't currently seeing lint errors in our PRs. You caught the console log issue and that wasn't showing up either in the PR.